### DISCUSSION TITLE
Relay-Server in config not necessary?

### DISCUSSION THREAD

**Discussion by ukdt7** [rustdesk/rustdesk#5783] [General]
### Bug Description

Hello, if i copy the network configuration with the rustdesk application. i get a base64 string that works with rustdesk.exe --config %configstring% BUT it doesnt fill in the field of the relay-server. i thought under this circumstance the app should not be able to connect to to another client or even the server. i figured out you can input whatever you want. it works fine. 

### How to Reproduce

copy the networkconfig from the rustdesk client an then run rustdesk.exe --config. it doesnt paste the relayserver

if you paste it directly in the client, it pastes the relayserver, too.

### Expected Behavior

it should paste the relay server too. i expected that it should work without the relay server defined in the config. i tried to find out if i connects to a default server, but it connects to the correct server and the connection to a remote client was established perfectly fine.

### Operating system(s) on local side and remote side

Win10 22h2

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

i can send you screenshots if you have questions

**Comment by ramon-flashtec** [comment]
Same here, using the official deployment Scripts, but I cannot connect to the other clients as long as the relayserver is missing. I have to insert the relayserver manually on each client to connect.
Not usable for hundreds of Clients :)

**Comment by T1MOXA** [comment]
Any update on this issue ?

**Comment by ramon-flashtec** [comment]
Still the same with version 1.2.3
I'm not able to deploy the client with relay-config.

**Comment by AH-Sass** [comment]
I just got my company to buy a pro license a couple weeks ago, but this issue is hurting my ability to deploy.
It does seem like this should have a straightforward fix.

**Comment by ramon-flashtec** [comment]
We want to switch to RustDesk too, but we have too much clients to do the setup manually.

**Comment by ukdt7** [comment]
Well, actually I dropped Rustdesk. I think the developers don‘t really care. The documentation is not really good, the application showed several unecpected behaviors, there are security issues in my opinion and it hurts while trying to deploy it. 

My conclusion is that RustDesk Pro is not a cent worth, when the free version is such a cathastrophe.

**Comment by Mike-Haan** [comment]
Same Issue here

**Comment by AH-Sass** [comment]
I don't know why this was converted from an issue last September, but I created a new issue for it.

https://github.com/rustdesk/rustdesk/issues/7117

**Comment by xlionjuan** [comment]
If Relay not specified in client, and `hbbs` (ID) server didn't provided relay (`hbbr`) server address

Relay server = ID server address
